gradle/script/artifact.gradle

```diff
@@ -15,8 +15,8 @@
  */
 
 ext {
-    _versionCode = 106
-    _versionName = "100.6.0"
+    _versionCode = 108
```

Comment:
How is version code used and why 108 not 1061 or something, for example?

Reply:
@TADraeseke - the version code is a positive integer to determine whether one version is more recent than another. More info here:
https://developer.android.com/studio/publish/versioning

The important thing is that it is bigger than the previous version code. It could have been 107 but since we already shipped 100.6.1 we cannot correct that anymore. Still, doesn't cause any issues being 108.

Reply:
Yeah I use it in the sample viewer releases too. OK 👍